ztools/http/client.go

```diff
@@ -613,6 +613,10 @@ func (c *Client) Do(req *Request) (resp *Response, err error) {
 					timeout: true,
 				}
 			}
+			if err == io.EOF {
```

Comment:
No, the HTTP lib returns a custom error message that includes the get and the ip address followed by io.EOF. I could parse the string in grabber.go to see if it contains io.EOF but this seemed easier.